app/modules/sale.js

```diff
@@ -77,7 +77,7 @@ export const participateInSale = (
   const scriptHashAddress = wallet.getAddressFromScriptHash(formattedScriptHash)
 
   const intents = [[ASSETS.NEO, neoToMint], [ASSETS.GAS, gasToMint]]
-    .filter(([symbol, amount]) => amount > 0)
+    .filter(([symbol, amount]) => amount > 0) // eslint-disable-line
```

Comment:
Why was this added here? Can we specify what rule is actually being disabled

Reply:
`symbol` was not used, the rule is no-unused-vars, but it would probably change the logic when you leave it out

Reply:
it actually did